recipe/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
   sha256: da1a945648679b21ba56b454b81e939451dc7951d9beb3c3e14f18f64dde6972
 
 build:
-  number: 1
+  number: 2
```

Comment:
You probably don't need to update the build number here

Reply:
Reverted build number to original value